second_2024_backend/425-questions-about-maximum/README.md

```
@@ -0,0 +1,70 @@
+
+Ограничение памяти
+
+256 МБ
+
+## Пример 1
+
+Ввод  | Вывод
+------|--------
+4 3   | 4
+1 3 1 | 1 2 3 4
+2 4 2 |
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)